Cargo.lock

```diff
@@ -9539,3 +9568,8 @@ dependencies = [
  "thiserror",
  "time 0.1.45",
 ]
+
+[[patch.unused]]
+name = "arbitrary"
+version = "1.2.2"
+source = "git+https://github.com/michaelsproul/arbitrary?rev=a572fd8743012a4f1ada5ee5968b1b3619c427ba#a572fd8743012a4f1ada5ee5968b1b3619c427ba"
```

Comment:
Ah looks like my arbitrary hacks are broken because of the latest release of `derive_arbitrary`. Unfortunately we still need a hacky version because of this issue: https://github.com/rust-lang/rust-clippy/issues/10185

I'll cook up a new version

Reply:
Sorry seems like I had committed a stale lockfile. Pushed a new commit now